.changeset/fresh-threads-wait.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/code-sdk': patch
+---
+
+Added a local boot option to defer initial thread creation until the client selects or creates a thread.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)